installer/PowerToysSetup/Product.wxs

```diff
@@ -247,6 +247,23 @@
               <Directory Id="ColorPickerResourcesFolder" Name="Resources"/>
             </Directory>
             <Directory Id="LauncherInstallFolder" Name="launcher">
+				<!-- Resource file directories -->
+				<?foreach Language in cs;de;es;fr;hu;it;ja;ko;nl;pl;pt-BR;pt-PT;ru;sv;tr;zh-Hans;zh-Hant?>
+				<!--NB: Ids can't contain hyphens-->
+				<?if $(var.Language) = pt-BR?>
+				<?define IdSafeLanguage = pt_BR?>
+				<?elseif $(var.Language) = pt-PT?>
+				<?define IdSafeLanguage = pt_PT?>
+				<?elseif $(var.Language) = zh-Hans?>
+				<?define IdSafeLanguage = zh_Hans?>
+				<?elseif $(var.Language) = zh-Hant?>
+				<?define IdSafeLanguage = zh_Hant?>
+				<?else?>
+				<?define IdSafeLanguage = $(var.Language)?>
+				<?endif?>
+				<Directory Id="LauncherRes$(var.IdSafeLanguage)Folder" Name="$(var.Language)" />
+				<?undef IdSafeLanguage?>
+				<?endforeach?>
```

Comment:
This block needs to be defined under each directory, so we have to check if there's a way to parameterize this for all modules